pennylane/devices/default_mixed.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
Could import `DefaultMixed` in `devices.__init__`, and also add to the autosummary there so that this shows up in the docs (although, maybe this is waiting for a follow up PR?)

Reply:
Would it make sense to make a PR into a holding branch, which we then merge into `master` once ready? I think this was the approach @co9olguy took for `DefaultTensor`.

Reply:
> I think this was the approach @co9olguy took for DefaultTensor

Just a word of warning with holding branches -- be prepared to either merge in changes from master _often_, or you will have a lot of hard to fix merge conflicts (that will require tracking down the original PR authors to work out what needs to be kept!)

From experience, I would caution against holding branches if possible.

Exceptions that would warrant a holding branch:

* We want to avoid piecemeal changes to the codebase due to an upcoming release

* The tests will not pass until the series of PRs are complete. But this should be a last resort, ideally there should not be tests failing at intermediate steps!

Reply:
Thanks @josh146 ! I'll follow your words of caution and avoid using a holding branch